src/lib.rs

```diff
@@ -1,3 +1,5 @@
+#[global_allocator]
+static GLOBAL: mimallocator::Mimalloc = mimallocator::Mimalloc;
```

Comment:
This globally changes the allocator? Shouldn't people do this in their applications rather than this library doing it?